src/components/landing/layout/footer/language-picker.tsx

```diff
@@ -11,7 +13,14 @@ const LanguagePicker: React.FC = () => {
     }
 
     return (
-        <select className="mb-2" value={i18n.language} onChange={onChangeLang}>
+        <Form.Control
+            as="select"
+            size="sm"
+            id="language-picker"
```

Comment:
I wouldn't use a static width. This could truncate new language names. How about "w-auto"?